README.md

```diff
@@ -446,6 +446,8 @@ For information about the governance of the Node.js project, see
   **Rich Trott** <<rtrott@gmail.com>> (he/him)
 * [vdeturckheim](https://github.com/vdeturckheim) -
   **Vladimir de Turckheim** <<vlad2t@hotmail.com>> (he/him)
+* [vitpavlenko](https://github.com/vitpavlenko) -
+  **Vitaliy Pavlenko** <<vitaliypavlenko01@gmail.com>> (he/him)
```

Comment:
```suggestion
```
Unrelated change, first-time contributors aren't added as collaborators right away.

You might be looking for the `AUTHORS` file at the root of the project to add yourself to, but that file shouldn't manually be changed as new contributors are automatically added to that file.

Reply:
okay